packages/charts/src/chart_types/xy_chart/rendering/rendering.bands.test.ts

```diff
@@ -70,84 +68,11 @@ describe('Rendering bands - areas', () => {
           value: { points },
         },
       ] = areas;
-      expect(points.length).toBe(4);
-      expect(points[0]).toEqual(
-        MockPointGeometry.default({
-          x: 0,
-          y: 80,
-          color: 'red',
-          seriesIdentifier: MockSeriesIdentifier.fromSpec(pointSeriesSpec),
-          value: {
-            accessor: 'y0',
-            x: 0,
-            y: 2,
-            mark: null,
-            datum: [0, 2, 10],
-          },
-          transform: {
-            x: 25,
-            y: 0,
-          },
-        }),
-      );
-
-      expect(points[1]).toEqual(
-        MockPointGeometry.default({
-          x: 0,
-          y: 0,
-          color: 'red',
-          seriesIdentifier: MockSeriesIdentifier.fromSpec(pointSeriesSpec),
-          value: {
-            accessor: 'y1',
-            x: 0,
-            y: 10,
-            mark: null,
-            datum: [0, 2, 10],
-          },
-          transform: {
-            x: 25,
-            y: 0,
-          },
-        }),
-      );
-      expect(points[2]).toEqual(
-        MockPointGeometry.default({
-          x: 50,
-          y: 70,
-          color: 'red',
-          seriesIdentifier: MockSeriesIdentifier.fromSpec(pointSeriesSpec),
-          value: {
-            accessor: 'y0',
-            x: 1,
-            y: 3,
-            mark: null,
-            datum: [1, 3, 5],
-          },
-          transform: {
-            x: 25,
-            y: 0,
-          },
-        }),
-      );
-      expect(points[3]).toEqual(
-        MockPointGeometry.default({
-          x: 50,
-          y: 50,
-          color: 'red',
-          seriesIdentifier: MockSeriesIdentifier.fromSpec(pointSeriesSpec),
-          value: {
-            accessor: 'y1',
-            x: 1,
-            y: 5,
-            mark: null,
-            datum: [1, 3, 5],
-          },
-          transform: {
-            x: 25,
-            y: 0,
-          },
-        }),
-      );
+      expect(points).toHaveLength(4);
+      expect(points[0]).toMatchSnapshot();
+      expect(points[1]).toMatchSnapshot();
+      expect(points[2]).toMatchSnapshot();
+      expect(points[3]).toMatchSnapshot();
```

Comment:
Can't we simplify this? I don't see a reason for individual snapshots.

```suggestion
      expect(points).toMatchSnapshot();
```

Reply:
done in https://github.com/elastic/elastic-charts/pull/1220/commits/46aa1c830ba048346a9e4a4fb1e36d38512ef59f